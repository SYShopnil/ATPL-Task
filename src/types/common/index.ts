export interface ICommonReturnData {
  message: string;
  status: number;
}

export enum EAuth {
  AuthTokenCookieName = "auth",
}
