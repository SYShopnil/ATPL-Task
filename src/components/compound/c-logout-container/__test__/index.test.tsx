import { render, screen } from "@testing-library/react";
import { CIsLoggedInContainer } from "..";
import { EDataTestId } from "@src/types/common";

describe("When Logout Container rendered ", () => {
  it("Expect it should render successfully", async () => {
    const { getByTestId } = render(
      <CIsLoggedInContainer isLoggedIn={true}>
        <div>Hello world</div>
      </CIsLoggedInContainer>
    ); // ARRANGE
    const myElement = getByTestId(EDataTestId.cLogoutContainer);
    // const myElement = screen.getByTestId(EDataTestId.cLogoutContainer);
    expect(myElement).toMatchSnapshot();
  });
});
