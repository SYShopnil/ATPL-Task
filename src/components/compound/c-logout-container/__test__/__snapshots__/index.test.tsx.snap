// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`When Logout Container rendered  Expect it should render successfully 1`] = `
<div
  data-testid="cLogoutContainer"
>
  <div>
    Hello world
  </div>
</div>
`;
